### PR TITLE
Fix message counter and some improvements

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Form/BaseEventCommandType.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('capacity', IntegerType::class, [
                 'required' => false,
+                'attr' => ['min' => 1],
```
Comment (thomasbisignani) [inline src/Form/BaseEventCommandType.php]: Why not using also a back validation for the min ?

File: templates/committee_manager/_space_layout.html.twig [hunk 2]
```diff
@@ -25,16 +22,55 @@
...
                 </li>
 
-                <li class="{{ is_active_route(app.request, ['app_committee_manager_list_members', 'app_committee_demote_host', 'app_committee_promote_host']) ? 'active' : '' }}">
-                    <a href="{{ path('app_committee_manager_list_members', {slug: committee.slug}) }}">Mes adhérents</a>
-                </li>
-
-                {% if is_canary_enabled() %}
-                    <li class="{{ is_active_route(app.request, 'app_message_committee_*') ? 'active' : '' }}">
-                        <a href="{{ path('app_message_committee_list', {committee_slug: committee.slug}) }}">Mes messages</a>
+                {% if committee.isApproved() %}
```
Comment (thomasbisignani) [inline templates/committee_manager/_space_layout.html.twig]: To be consistent with the others calls in the project, you can just write `committee.isApproved` without parenthesis

Like line 68

File: templates/committee_manager/dashboard.html.twig [hunk 3]
```diff
@@ -42,11 +42,13 @@
 {% endblock %}
 
 {% block committee_block %}
-    <div class="pre-space__item-card">
+    <div class="pre-space__item-card {{ not committee.isApproved() ? 'pre-space__item-card--muted' : '' }}">
```
Comment (thomasbisignani) [inline templates/committee_manager/dashboard.html.twig]: Same here, and line 49

File: src/Form/BaseEventCommandType.php (hunk 1 above)
Comment (ottaviano) [reply src/Form/BaseEventCommandType.php]: done already https://github.com/EnMarche/en-marche.fr/commit/dec74a027f142af5598853249f273771f8fe684f#diff-6f883ec988ade37a236054f951485041R26

File: templates/committee_manager/_space_layout.html.twig (hunk 2 above)
Comment (ottaviano) [reply templates/committee_manager/_space_layout.html.twig]: done

File: templates/committee_manager/dashboard.html.twig (hunk 3 above)
Comment (ottaviano) [reply templates/committee_manager/dashboard.html.twig]: done

**Pull request merged**